test/coverage-test/test/coverage.test.ts

```diff
@@ -27,3 +30,11 @@ test('import meta env', () => {
 test('cover function counts', () => {
   expect(second()).toBe(2)
 })
+
+test.skipIf(isBrowser)('run dynamic ESM file', async () => {
+  await runDynamicFileESM()
+})
+
+test.skipIf(isBrowser)('run dynamic CJS file', async () => {
```

Comment:
Noticed that browser mode crashes when these dynamic files are imported. Even when coverage is disabled.